recipes/rb-escape_utils/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "escape_utils" %}
+{% set version = "1.3.0" %}
+
+package:
+  name: rb-{{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://rubygems.org/downloads/{{ name }}-{{ version }}.gem
+    sha256: dffb7010922880ace6ceed642156c64e2a64620f27e0849f43bc4f68fd3c2c09
+
+build:
+  number: 0
+  script:
+    - gem install -N -l -V --norc --ignore-dependencies {{ name }}-{{ version }}.gem
+    - gem unpack {{ name }}-{{ version }}.gem
+    - make -C {{ PREFIX }}/lib/ruby/gems/{{ ruby }}.0/gems/{{ name }}-{{ version }}/ext/{{ name }} clean
```

Comment:
@svengiegerich, you call `make` here. where did you get these build instructions from and what happens when you remove this line?